spec/compiler/semantic/doc_spec.cr

```diff
@@ -62,6 +62,19 @@ describe "Semantic: doc" do
     bar.doc.should eq("Hello")
   end
 
+  it "stores doc for const when using ditto" do
+    result = semantic %(
+      # A number
+      ONE = 1
+
+      # ditto
```

Comment:
```suggestion
      # :ditto:
```
Both syntax are valid, but without colons will eventually go away with https://github.com/crystal-lang/crystal/pull/6362

Reply:
In that case also page https://crystal-lang.org/reference/conventions/documenting_code.html should be updated to describe both ":ditto:" and "ditto" (currently it only mentions "ditto" without colons).